src/tests.ts

```diff
@@ -78,30 +82,53 @@ async function deployContracts(arweave, walletA, walletB) {
     settings: [["isTradeable", true]],
   };
 
-  const contractA = await createContract(
-    arweave,
-    walletA.jwk,
-    contractSrc,
-    JSON.stringify(initialStateA)
-  );
+  let contractATxId, contractBTxId;
 
-  const contractB = await createContract(
-    arweave,
-    walletA.jwk,
-    contractSrc,
-    JSON.stringify(initialStateB)
-  );
+  ({ contractTxId: contractATxId } = await warp.createContract.deploy({
+    wallet: walletA.jwk,
+    initState: JSON.stringify(initialStateA),
+    src: contractSrc,
+  }));
 
-  await mine();
+  ({ contractTxId: contractBTxId } = await warp.createContract.deploy({
+    wallet: walletB.jwk,
+    initState: JSON.stringify(initialStateB),
+    src: contractSrc,
+  }));
+
+  await warp.testing.mineBlock();
+
+  const contractA = warp
+    .contract<StateInterface>(contractATxId)
+    .setEvaluationOptions({
+      internalWrites: true,
```

Comment:
no that I look and the contract's code - it doesn't even use the internal writes feature - so this flag can be removed/switched to false....

Reply:
ah, it's this `@verto/flex` lib that is using it, my mistake!